tests/robustness/model/non_deterministic.go

```diff
@@ -71,11 +71,10 @@ type EtcdNonDeterministicResponse struct {
 
 func (states nonDeterministicState) Step(request EtcdRequest, response EtcdNonDeterministicResponse) (bool, nonDeterministicState) {
 	if len(states) == 0 {
-		// states were not initialized
-		if response.Err != nil || response.ResultUnknown || response.Revision == 0 {
```

Comment:
Print a log if `response.Err != nil || response.ResultUnknown`?

Reply:
This is model code which is used for linearization. This means that this is an expected path that could be potentially called hundreds of times. Don't think we should have any logging in model.